docs/linux/reporting_kernel_bugs.md

```diff
@@ -1,11 +1,39 @@
 # Reporting Linux kernel bugs
 
-Before reporting a bug make sure nobody else already reported it. The easiest way to do this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller), [syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs) and [syzbot dashboard](https://syzkaller.appspot.com/upstream) for key frames present in the kernel stack traces.
-
-Please report found bugs to the Linux kernel maintainers.
+Before reporting a bug **make sure nobody else has already reported it**. The easiest way to do
+this is to search through the [syzkaller mailing list](https://groups.google.com/forum/#!forum/syzkaller),
+[syzkaller-bugs mailing list](https://groups.google.com/forum/#!forum/syzkaller-bugs),
+[syzbot dashboard](https://syzkaller.appspot.com/upstream),
+and [kernel mailing lists](https://lore.kernel.org/) for key frames present in the kernel stack traces.
+
+**Do NOT report bugs on old kernel versions**.
+Old kernel generally means older than a week. Reports on old kernels are usually negatively accepted,
```

Comment:
"Reports on old kernels are usually negatively accepted" -> "Reports on old kernels are typically not considered as valid, and asked ..."

Reply:
Done